wesnoth-nr.po

```diff
@@ -2729,31 +2636,30 @@ msgstr "Tak se předveď!"
 
 #. [scenario]: id=05a_01_The_Pursuit
 #: data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:1252
-#, fuzzy
-#| msgid "Wow! There is at least 1,000 gold in here!"
 msgid "There are at least several hundred gold coins in here!"
-msgstr "Jupí! Tady je aspoň tisíc zlatých!"
+msgstr "Je tady aspoň několik set zlatých!"
 
 #. [scenario]: id=05a_01_The_Pursuit
 #: data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:1254
 msgid "I don’t think the undead will be be needing this anyway."
-msgstr ""
+msgstr "Nemrtví je asi stejně nebudou potřebovat."
 
 #. [scenario]: id=05a_01_The_Pursuit
 #: data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:1256
 msgid ""
 "Our people could rebuild their lives ten-fold with the gold in this treasury."
 msgstr ""
+"Naši lidé mohou ze zlata v této pokladně desetinásobně obnovit své živobytí."
 
 #. [scenario]: id=05a_01_The_Pursuit
 #: data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:1258
 msgid "We’re rich!"
-msgstr "Jsme bohatí!!"
+msgstr "Jsme bohatí!"
 
 #. [scenario]: id=05a_01_The_Pursuit
 #: data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:1260
 msgid "Why do these skeletons have so much gold just sitting around?"
-msgstr ""
+msgstr "Proč mají ty kostlivci tolik zlata, když na něm jen sedí?"
```

Comment:
anebo "maj ty", pokud to má být hovorově
```suggestion
msgstr "Proč mají ti kostlivci tolik zlata, když na něm jen sedí?"
```

Reply:
I přesto, že z katalogů není zcela zřejmé, kdo to říká (je tam nějaká chybka), tak tipuji, že to je obecně nějaká jednotka. Proto jsem se přiklonil k hovorovému tvaru.